server/sonar-server/src/main/java/org/sonar/server/user/ws/GroupsAction.java

```diff
@@ -78,7 +77,7 @@ public void define(NewController context) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkLoggedIn().checkPermission(GlobalPermissions.SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not 'Administer System' permission... or is that the name we give to root?